### PR TITLE
[release-1.0.0-RC3] Adapt fixtures new rights

### PR DESCRIPTION
**Pull request by ballinette**
see  #59

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,26 +1,25 @@
-kuzzle:
-  image: kuzzleio/kuzzle
+loadbalancer:
```
Comment (scottinet) [inline docker-compose.yml]: shouldn't this be named "proxy"?

**Comment by scottinet** [comment]
+1

File: docker-compose.yml [hunk 2]
```diff
@@ -1,26 +1,25 @@
...
 
-kuzzleBo:
-  image: kuzzleio/bo:alpine
+bo:
+  image: kuzzleio/bo:develop
   ports:
     - "3000:3000"
   links:
-    - kuzzle
+    - loadbalancer:kuzzle
```
Comment (dbengsch) [inline docker-compose.yml]: an idea was to rename "kuzzle" in "api", to avoid such misleading container bindings

**Comment by benoitvidis** [comment]
+1

**Comment by xbill82** [comment]
So, guys do we stick to `proxy` or `api`? I'd go for `proxy`.

File: docker-compose.yml (hunk 2 above)
Comment (xbill82) [inline docker-compose.yml]: I'd rather leave `kuzzle` here, since the Proxy is the actual façade for Kuzzle against tiers applications.

**Comment by ballinette** [comment]
+1

**Pull request merged**